sectionproperties/pre/sections.py

```diff
@@ -27,25 +31,51 @@ class Geometry:
     :vartype perimeter: list[int]
     """
 
-    def __init__(self, control_points, shift):
-        """Inits the Geometry class."""
+    # Design questions: 
+    # 1. Should a Geometery instance by immutable? Should changes to the underlying
+    #    shapely geometry return a new Geometry instance instead of mutating self.geom?
```

Comment:
Thanks! That was an earlier comment I made to myself, mostly. I have developed a penchant for working with immutable objects but they are a pain to setup and can cause unexpected errors in expected behaviour. I think it would makes sense to keep them mutable. Yes, this means that the shapely geometry can be altered without needing new Geometry instances.